src/store.c

```diff
@@ -622,34 +800,33 @@ struct rill_pairs *rill_store_scan_vals(
     return NULL;
 }
 
-size_t rill_store_dump_vals(
-        const struct rill_store *store, rill_val_t *out, size_t cap)
-{
-    size_t len = cap < store->vals->len ? cap : store->vals->len;
-    memcpy(out, store->vals->data, len * sizeof(*out));
-    return len;
-}
-
 size_t rill_store_dump_keys(
```

Comment:
since you renamed the `rill_store_keys` to `rill_store_keys_count` then you should rename this function to `rill_store_keys`.

Reply:
Fixorzed.